demo/compositions/useArtifactsMock.ts

```diff
@@ -0,0 +1,45 @@
+import { useSeeds } from './useSeeds'
+import { Artifact } from '@/models'
+import { mocker } from '@/services'
+import { coinflip, repeat } from '@/utilities'
+
+export function useArtifactMock(override?: Partial<Artifact>, useTaskRun: boolean = coinflip(0.5)): Artifact {
```

Comment:
Am I missing something or is the override not used anywhere

Reply:
Nah you're right, forgot to pass it to the artifact, nice catch (I'm surprised I didn't get a warning about that, these files might be ignored)